test_vm/tests/test_vm_test.rs

```diff
@@ -129,66 +126,6 @@ fn test_sent() {
     v.assert_state_invariants();
 }
 
-fn assert_embryo_actor(exp_bal: TokenAmount, v: &VM, addr: Address) {
```

Comment:
I don't see where these are moving too so I'm suspicious they are being deleted and they look very nice and should stay